### PR TITLE
Editor Compilation Warning Cleanup

### PR DESCRIPTION
**Pull request by daalta**
### Purpose
* Fixed some Unity Editor Warnings.
* This should hopefully make it easier to see if a PR causes new warnings.
* Continuation of https://github.com/unitystation/unitystation/pull/3477
* Progress towards #2526 
### Notes:
* Removed unused variales.
  * Some variables were commented out, left in or left with a TODO comment. Most were simply removed.
* Explicitly initialized bool, nullable and int variables with their default value (false, null or 0).
* Commented out unreachable code and added TODOs.
* Ignored some false "unused variable" warnings (which are being used in the inspector).
* Deleted some redundant includes.
* Deleted missing prefab references.
* Rider also did some autoformatting, deleting some spaces.

There's still about 80 warning left after compiling (such as https://github.com/unitystation/unitystation/issues/3875). Let's try and get those to 0, preferrably without ignoring all warnings :)

### Checks

- [x] Code is sufficiently commented
- [x] Code is indented with tabs and not spaces
- [x] The PR does not include any unnecessary .meta, .prefab or **.unity (scene) changes**
- [x] The PR does not bring up any new compile errors (HAH)
- [x] The PR has been tested in editor

### CODE REVIEW

**Comment by CodeKalias** [comment]
>d87236c49b16e98cb75331449a74bf8f30d6829b
Deleted empty references in some SpawnableList

These are `Fire Extinguisher.prefab` if you'd like to re-add them

File: UnityProject/Assets/Scripts/Variable Viewer/BookViewer/TestVariableViewerScript.cs [hunk 1]
```diff
@@ -8,7 +8,6 @@ public class TestVariableViewerScript : MonoBehaviour
 	public bool Pbool = true;
 	public int Pint = 55;
 	public string pstring = "yoyyyoy";
-	private string PVstring = "My porn collection"; //Me private variables!!!
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Variable Viewer/BookViewer/TestVariableViewerScript.cs:11]: awww man,not my collection xD

**Comment by daalta** [comment]
> 
> 
> > [d87236c](https://github.com/unitystation/unitystation/commit/d87236c49b16e98cb75331449a74bf8f30d6829b)
> > Deleted empty references in some SpawnableList
> 
> These are `Fire Extinguisher.prefab` if you'd like to re-add them

Sure, I'll add them. One into each cabinet?

**Comment by CodeKalias** [comment]
> > > [d87236c](https://github.com/unitystation/unitystation/commit/d87236c49b16e98cb75331449a74bf8f30d6829b)
> > > Deleted empty references in some SpawnableList
> > 
> > 
> > These are `Fire Extinguisher.prefab` if you'd like to re-add them
> 
> Sure, I'll add them. One into each cabinet?

Four into EngineeringFire.asset
Two into Fire.asset

**Comment by daalta** [comment]
@CodeKalias Done!

**Comment by Bod9001** [comment]
all good to merge now?

**Comment by daalta** [comment]
@Bod9001 Unless there's another issue, sure.

My PR affects a lot of files. If Rider or Unity missed something, then there's suddenly a variable missing 😋

That... uh... worries me a bit!

**Comment by Bod9001** [comment]
cool cool

**Pull request merged**